app/src/app/app.component.ts

```diff
@@ -13,10 +13,20 @@
 // limitations under the License.
 
 import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ModalComponent } from './components/specmath-modal/modal.component';
+
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
+  constructor(public dialog: MatDialog) {
```

Comment:
since this value is never changed, it should be marked `readonly`.

google typescript style guide also mentions:
> TypeScript symbols are public by default. Never use the public modifier except when declaring non-readonly public parameter properties (in constructors).

```
class Foo {
  bar = new Bar();  // GOOD: public modifier not needed

  constructor(public baz: Baz) {}  // public modifier allowed
}
```
it should not have the public keyword since it's readonly

Reply:
noted 👍